aten/src/ATen/native/native_functions.yaml

```diff
@@ -8224,6 +8224,23 @@
   use_c10_dispatcher: full
   variants: function, method
 
+- func: _syevd_helper(Tensor self, bool compute_v, str uplo) -> (Tensor, Tensor)
```

Comment:
Could you please replace the parameter names with something more common, such as `compute_v -> compute_eigenvectors`, `uplo -> upper`.